java/test/src/main/java/org/ray/api/test/BaseTest.java

```diff
@@ -24,7 +24,11 @@ public void setUpBase(Method method) {
     // These files need to be deleted after each test case.
     filesToDelete = ImmutableList.of(
         new File(Ray.getRuntimeContext().getRayletSocketName()),
-        new File(Ray.getRuntimeContext().getObjectStoreSocketName())
+        new File(Ray.getRuntimeContext().getObjectStoreSocketName()),
+        // TODO(pcm): This is a workaround for
+        // https://github.com/ray-project/ray/pull/5450
+        // and should be fixed properly.
```

Comment:
The issue should be described. The link just points to this PR, not a description of the issue.